### PR TITLE
feat(authentication): add currentProjectId in AuthenticationFacade

### PR DESCRIPTION
**Pull request by CHLK**
#### What type of PR is this?
type-feature

#### What this PR does / why we need it:
add currentProjectId in AuthenticationFacade
#### Which issue(s) this PR fixes:

Fixes #

#### Additional documentation e.g., usage docs, etc.:

```docs

```

### CODE REVIEW

File: server/starters/web-starter/src/main/java/com/oceanbase/odc/service/iam/auth/WebAuthenticationFacade.java [hunk 1]
```diff
@@ -24,5 +25,6 @@
  */
 @Profile("alipay")
 @Service
+@ConditionalOnMissingBean(AuthenticationFacade.class)
```
Comment (yhilmare) [inline server/starters/web-starter/src/main/java/com/oceanbase/odc/service/iam/auth/WebAuthenticationFacade.java:28]: we may use some properties to control this bean's init

Comment (CHLK) [reply server/starters/web-starter/src/main/java/com/oceanbase/odc/service/iam/auth/WebAuthenticationFacade.java:28]: okay, i'll  make this change

**Review by yhilmare — APPROVED** [review]
lgtm

**Review by PeachThinking — APPROVED** [review]
lgtm

**Pull request merged**